### ISSUE TITLE
Multiplier complexity analysis based on array length

### ISSUE THREAD

**Issue by tvvignesh** [pa-bru/graphql-cost-analysis#8]
I have a schema like this:
```
type PayrollType {
  mou: String!
  eou: String!
  ecode: String!
  payElements(filter: [String]): [PayElementType]
}
```

Here, I want to calculate the cost of payElements attribute where I pass the filter as an array of payelements for which I want to return the data like this: payElements(filter: ["ELEM1","ELEM2"])

So, here I want to base the multiplier based on length but as far as I know the filter can contain only integers as per this library. Any way to calculate multiplier cost by array length?

Here, the cost will be complexity*2 where 2 is the length of the array.

And before I forget to tell this, amazing work with the library. It works like a charm.

**Comment by pa-bru** [comment]
Hello @tvvignesh, I added the possibility to take the length of a parameter of type array for the multiplier value. This is now available in the v1.0.1. 

So you can achieve that directly by passing the multipliers attribute like this: 
```
@cost (multipliers: ["filter"])
```

This is also available with the costMap option. 

:)

**Comment by tvvignesh** [comment]
Thanks for that. That was quick and awesome. Will try it out.
